dist/index.html

```diff
@@ -1,12 +1,37 @@
 <!DOCTYPE html>
```

Comment:
- [ ] Kindly use Github Flow properly. You are merging your main branch to interactive list branch. You supposed to merge all your feature branches in this case your interactive list branch into main branch to follow Github Flow properly :+1: